Marlin/Configuration.h

```diff
@@ -1485,6 +1494,11 @@
   #define U8GLIB_SH1106
 #endif
 
+//#define CR10_stockdisplay
+
+// MKS OLED 1.3" 128 × 64 FULL GRAPHICS CONTROLLER 
+#define MKS_OLED13_128x64_FULL_GRAPHICS_CONTROLLER
```

Comment:
As mentioned, it's nice to have a link to the display's product page, if possible.